test/e2e/tinkerbell_test.go

```diff
@@ -470,6 +502,45 @@ func TestTinkerbellUpgrade125MulticlusterWorkloadClusterWorkerScaleupWithAPI(t *
 	)
 }
 
+func TestTinkerbellUpgrade126MulticlusterWorkloadClusterWorkerScaleupWithFluxAPI(t *testing.T) {
+	provider := framework.NewTinkerbell(t, framework.WithUbuntu126Tinkerbell())
+	managementCluster := framework.NewClusterE2ETest(
+		t,
+		provider,
+		framework.WithClusterFiller(
+			api.WithKubernetesVersion(v1alpha1.Kube126),
+			api.RemoveAllWorkerNodeGroups(),
+		),
```

Comment:
I would probably move this to the `WithClusterConfig()` using the `api.ClusterToConfigFiller` method. Modifying the config through both ClusterE2ETest opts and `WithClusterConfig` can have unintended consequences. I think `WithClusterConfig()` will overwrite the cluster fillers you add here.

Reply:
Thanks, I think `WithClusterConfig` will try and [append to any existing clusterFillers](https://github.com/aws/eks-anywhere/blob/50ee3e1e0705b769039fdff04d53e471e198dd50/test/framework/cluster.go#L566-L567) but definitely agree it'd be best to group them.